controlplane/src/db/schema.ts

```diff
@@ -658,6 +742,60 @@ export const organizationMemberRolesRelations = relations(organizationMemberRole
   }),
 }));
 
+export const organizationFeatures = pgTable(
+  'organization_features',
+  {
+    id: uuid('id').notNull().primaryKey().defaultRandom(),
+    organizationId: uuid('organization_id')
+      .notNull()
+      .references(() => organizations.id, {
+        onDelete: 'cascade',
+      }),
+    feature: text('feature').notNull(),
+    enabled: boolean('enabled').default(true),
+    limit: integer('limit'),
+  },
+  (t) => {
+    return {
+      nameIndex: uniqueIndex('organization_feature_idx').on(t.organizationId, t.feature),
+    };
+  },
+);
+
+/**
+ * @deprecated
+ * Use organizationFeatures instead
```

Comment:
Can we avoid deprecating stuff and do the migrations in this PR?

Reply:
We can remove it after manually migrating existing customers, but i left it here so the table isn't dropped.